tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace_tests/test_gtir_to_sdfg.py

```diff
@@ -170,6 +170,213 @@ def test_gtir_copy_self():
     assert np.allclose(a, ref)
 
 
+def test_gtir_tuple_swap():
+    domain = im.call("cartesian_domain")(
+        im.call("named_range")(gtir.AxisLiteral(value=IDim.value), 0, "size")
+    )
+    testee = gtir.Program(
+        id="gtir_tuple_swap",
+        function_definitions=[],
+        params=[
+            gtir.Sym(id="x", type=IFTYPE),
+            gtir.Sym(id="y", type=IFTYPE),
+            gtir.Sym(id="size", type=SIZE_TYPE),
+        ],
+        declarations=[],
+        body=[
+            gtir.SetAt(
+                expr=im.make_tuple("y", "x"),
+                domain=domain,
+                target=im.make_tuple("x", "y"),
```

Comment:
- I forgot, but I guess we concluded that we want to support this pattern?
- Can you add a TODO to add a frontend check for this case.

Reply:
Yes, I think we said that in theory we should support it. I've added the TODO comment.